src/applications/static-pages/covid-vaccine-updates-cta/createCovidVaccineUpdatesWidget.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-export default function createCovidVaccineUpdatesWidget(store, widgetType) {
-  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
-  if (root) {
+export default function createCovidVaccineUpdatesWidget(store, _widgetType) {
+  const isCovidVaccineUpdatesPage =
+    document.location.pathname === '/health-care/covid-19-vaccine/';
```

Comment:
Not sure if you have a constants file we could put this pathname in but might be nice! 🙂

Reply:
I can move it outside of the function in a follow-up PR but personally I would rather keep that path local to the widget (unless it starts being required in more than one place)

Reply:
I'm always a fan of abstracting later once it's needed, I dig it! 😎